src/index.html

```diff
@@ -6,8 +6,41 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <title>Product cards</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
 </head>
 <body>
-  <h1>Product cards</h1>
+  <div class="product-card" data-qa="card">
+    <img src="./images/imac.jpeg" class="product-card__image" alt="imac-image">
+    <div class="product-card__name">
+      Моноблок APPLE A1419 iMac
+      27" Retina 5K (MNED2UA/A)
+    </div>
+    <div class="product-card__code">
+      Код товара: 195434
+    </div>
+    <div class="product-card__rate">
+      <div class="product-rate__mark">
+        <div class="product-mark__star"></div>
+        <div class="product-mark__star"></div>
+        <div class="product-mark__star"></div>
+        <div class="product-mark__star"></div>
+        <div class="product-mark__star"></div>
+      </div>
+      <div class="product-rate__response">
+        Отзывов: 5
+      </div>
+    </div>
+    <div class="product-card__price-stroke">
+      <div class="price-stroke__price-name">
+        Цена:
+      </div>
+      <div class="price-stroke__price-number">
+        69 999 грн
+      </div>
+    </div>
+    <div class="product-card__select-buy" data-qa="hover">
```

Comment:
select-buy?
```suggestion
    <div class="product-card__button" data-qa="hover">
```
```suggestion
    <div class="product-card__link" data-qa="hover">
```
Это либо кнопка, либо ссылка, не див